genesyscloud/architect_flow/genesyscloud_architect_flow_init_test.go

```diff
@@ -1,16 +1,18 @@
 package architect_flow
 
 import (
-	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
 	"sync"
 	"testing"
-)
 
-// providerDataSources holds a map of all registered data sources
-var providerDataSources map[string]*schema.Resource
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/mypurecloud/platform-client-sdk-go/v125/platformclientv2"
+)
 
-// providerResources holds a map of all registered resources
-var providerResources map[string]*schema.Resource
+var (
+	sdkConfig           *platformclientv2.Configuration
```

Comment:
You'll need to call AuthorizeSdk in this file. You can find an example in the package `outbound_campaign`. In the outbound campaign package, `log.Fatalf` is called if `AuthorizeSdk` fails because the tests require that the config be authorized or else they won't pass. In this package, you're only using it for a cleanup function, so `log.Printf` `would suffice

Reply:
These changes are removed